sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocTypeInfo.cs

```diff
@@ -11,8 +11,5 @@ public partial class DocTypeInfo
     {
         /// <summary> Description of the document semantic schema. </summary>
         public IReadOnlyDictionary<string, DocumentFieldSchema> FieldSchema { get; }
-
-        /// <summary> Custom model build mode. </summary>
-        internal DocumentBuildMode? BuildMode { get; }
```

Comment:
As always, deleting it from here so CodeGen can generate it with the `public` access modifier.